routes/caniuse/lib/constants.ts

```diff
@@ -3,20 +3,26 @@ export const BROWSERS = new Map([
   ["and_ff", "Firefox (Android)"],
   ["and_uc", "UC Browser (Android)"],
   ["android", "Android"],
-  ["bb", "Blackberry"],
   ["chrome", "Chrome"],
   ["edge", "Edge"],
   ["firefox", "Firefox"],
-  ["ie", "IE"],
   ["ios_saf", "Safari (iOS)"],
-  ["op_mini", "Opera Mini"],
   ["op_mob", "Opera Mobile"],
   ["opera", "Opera"],
   ["safari", "Safari"],
   ["samsung", "Samsung Internet"],
 ]);
 
-export const DEFAULT_BROWSERS = ["chrome", "firefox", "safari", "edge"];
+export const DEFAULT_BROWSERS = [
+  "and_chr",
+  "and_ff",
+  "chrome",
+  "edge",
+  "firefox",
+  "ios_saf",
+  "safari",
+  "samsung",
+];
```

Comment:
This will break specs using old ReSpec versions.

Reply:
Just wondering how it would break? Wouldn't they just see more results by default or am I missing something?

Reply:
Yeah. More results by default (than documented for that ReSpec version) is a breakage.

Reply:
It might be nothing. We'll see from logs.

Maybe we should always send browsers list from ReSpec, so it's not tied to the API.

Reply:
The defaults are allowed to change without notice. But yeah, we should definitely move them to the client.

Reply:
Filed: https://github.com/w3c/respec/issues/4091 

I think we should be good here tho. Let's try to get this landed today or tomorrow.

Reply:
Result of logs for deprecated usage:

Duration: 2022-03-15T07:10:03.402Z - 2022-03-15T06:58:29.583Z
Count: 137

Unique:
```
2022-03-14T23:57:51.527Z | GET  200 /caniuse/?feature=high-resolution-time&versions=4&format=html | https://w3c.github.io/ | - | 5.447 ms | deprecated=true
2022-03-16T00:06:01.163Z | GET  200 /caniuse/?feature=link-rel-preload&versions=4&format=html | https://w3c.github.io/ | - | 1.055 ms | deprecated=true
2022-03-15T07:10:03.402Z | GET  200 /caniuse/?feature=netinfo&versions=4&browsers=chrome,firefox,and_ff,safari&format=html | https://wicg.github.io/ | - | 0.969 ms | deprecated=true
2022-03-16T00:05:07.269Z | GET  200 /caniuse/?feature=payment-request&versions=4&format=html | https://w3c.github.io/ | - | 1.086 ms | deprecated=true
2022-03-15T02:59:25.467Z | GET  200 /caniuse/?feature=pointer&versions=4&format=html | https://w3c.github.io/ | - | 1.587 ms | deprecated=true
2022-03-15T13:26:53.053Z | GET  200 /caniuse/?feature=resource-timing&versions=4&format=html | https://w3c.github.io/ | - | 1.283 ms | deprecated=true
2022-03-15T13:47:14.093Z | GET  200 /caniuse/?feature=web-app-manifest&versions=1&browsers=chrome,and_chr,edge,and_ff,ios_saf&format=html | https://w3c.github.io/ | - | 3.249 ms | deprecated=true
2022-03-17T06:02:46.621Z | GET  304 /caniuse/?feature=netinfo&versions=4&browsers=chrome,firefox,and_ff,safari&format=html | https://wicg.github.io/ | - | 1.203 ms | deprecated=true
```

Reply:
I still think it's safer to add the constants to the new route (and in frontend). Keep the old constants as long as we keep that route.

Reply:
I’m still not sure what’s unsafe about it? Showing the mobile support is never harmful and it doesn’t break rendering the spec. What am I missing?

Reply:
For example: 
https://github.com/w3c/payment-request/blob/d8ed8b303393343ea0efcbb9fda365e8a952eeb9/index.html#L70

It’s literally saying, “just show me whatever”. Same with the rest.

Reply:
https://respec.org/docs/#configuration-options-0 is saying "we'll show `["chrome", "firefox", "safari", "edge"]` by default". I'm saying changing the default is a breaking change.
I don't see any harm in adding new constants in new route and keep old route working as is.

Reply:
Updated docs. Just said the default is set automatically.

Reply:
Alright then.